addons/l10n_cn/__init__.py

```diff
@@ -2,3 +2,15 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 # Copyright (C) 2007-2014 Jeff Wang(<http://jeff@osbzr.com>).
+
+from odoo import api, SUPERUSER_ID
+
+
+def _auto_install_l10n_cn(cr, registry):
```

Comment:
useless post_init_hook: to remove

Reply:
@qdp-odoo 
> When I create a database and give the country 'China'  all the COA are installed.
- here is the case that for I need to use post_init_hook
> when I manually select the COA then It should only install the selected COA not all.
 - If I install l10n_cn => It should automatically install l10n_cn + l10n_cn_small_business
- If I install l10n_cn_small_business => It should automatically install  l10n_cn + l10n_cn_small_business
- If I install l10n_standard => It should automatically install l10n_cn + l10n_cn_standard